reducing_dims.py

```diff
@@ -1,6 +1,5 @@
 import numpy as np
 from scipy.linalg import orth
-import pandas as pd
 np.seterr(divide='ignore', invalid='ignore')

```

Comment:
Why do you need that? This kind of error indicates that you have broken assumptions somewhere in your code, what's the point of silencing it? You cannot find the error if you silenced it O.o